src/cleanup.ts

```diff
@@ -106,6 +106,8 @@ async function collectAndPublishBuildInfoIfNeeded() {
     // Publish the build info to Artifactory
     try {
         core.startGroup('Publish the build info to JFrog Artifactory');
+        // Used for usage reporting
+        core.exportVariable("JFROG_CLI_USAGE_BUILD_PUBLISHED_AUTO", "TRUE");
```

Comment:
Can we change it to `JFROG_CLI_USAGE_AUTO_BUILD_PUBLISH`?

Reply:
Sure, so mabye `AUTO_BUILD_PUBLISHED`, will also update in the core consts.